encoding/ccf/decode.go

```diff
@@ -797,6 +801,23 @@ func (d *Decoder) decodeWord64() (cadence.Value, error) {
 	return cadence.NewMeteredWord64(d.gauge, i), nil
 }
 
+// decodeWord128 decodes word128-value as
+// language=CDDL
+// word128-value = bigint .ge 0
+func (d *Decoder) decodeWord128() (cadence.Value, error) {
+	// NewMeteredWord128FromBig checks if decoded big.Int is positive.
+	return cadence.NewMeteredWord128FromBig(
+		d.gauge,
+		func() *big.Int {
+			bigInt, err := d.dec.DecodeBigInt()
+			if err != nil {
+				panic(fmt.Errorf("failed to decode Word128: %s", err))
```

Comment:
Could you please add a test case for this line?

Reply:
Done, added `TestDecodeWord128Invalid` in ccf_test.go